src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -61,9 +65,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(IcebergInputFormat.class);
 
   static final String TABLE_LOCATION = "location";
-  static final String TABLE_FILTER_SERIALIZED = "iceberg.filter.serialized";
+  static final String TABLE_FILTER_SERIALIZED = "hive.io.filter.expr.serialized";
```

Comment:
Should we just use this directly instead of creating our own? https://github.com/apache/hive/blob/c34ee9d79bf6931a92b61af98a6c8f09c6b9ad73/ql/src/java/org/apache/hadoop/hive/ql/plan/TableScanDesc.java#L96